tests/common_tests/helpers/generate_test_tp_model.py

```diff
@@ -30,6 +30,9 @@
 
 
 def generate_test_tp_model(edit_params_dict, name=""):
+    # Add "supported_input_activation_n_bits" to match "activation_n_bits" if not defined.
+    if 'activation_n_bits' in edit_params_dict and 'supported_input_activation_n_bits' not in edit_params_dict:
```

Comment:
use const from mct/constants
and add supported_input_activation_n_bits to consts as well